compiler_opt/rl/train_locally.py

```diff
@@ -126,7 +126,7 @@ def train_eval(agent_name=constant.AgentName.PPO,
       train_sequence_length=train_sequence_length)
 
   def sequence_example_iterator_fn(seq_ex: List[str]):
-    return iter(dataset_fn(seq_ex).repeat())
+    return iter(dataset_fn(seq_ex).repeat().prefetch(tf.data.AUTOTUNE))
```

Comment:
does where the prefetch is affect the performance?

Reply:
> is the performance now similar or slow to the ABCCBA solution?

about 2~3x slower, but now we're talking <1s (ABCCBA) vs 2~3s for this

Reply:
ic, yeah, i guess it makes a lot of sense to have the prefetch at the end of the dataset pipeline